docker/Real_device

```diff
@@ -69,9 +69,36 @@ RUN apt-get -qqy update && apt-get -qqy install --no-install-recommends \
     net-tools \
     ffmpeg \
     jq \
+    curl \
+    libavcodec-dev \
+    libavdevice-dev \
+    libavfilter-dev \
+    libavformat-dev \
```

Comment:
could you give more detail information as a comment like what kind of packages are those and why we need those?

Reply:
Its basically ffmpeg-dev, which is split in different packages. To be honest, we probably dont need all of them, but scrcpy build needed ffmpeg-dev, so i included all parts. Maybe i should check which parts are really used.

Reply:
Its basically ffmpeg-dev, which is split in different packages. To be honest, we probably dont need all of them, but scrcpy build needed ffmpeg-dev, so i included all parts. Maybe i should check which parts are really used.

Reply:
yes, I agreed that we need to check which component that we really need, let me know if you need anything from me.

Reply:
dependency('libavformat'),
        dependency('libavcodec'),
        dependency('libavutil'),

Are mentioned in the scrcpy meson file.